iroh/src/commands/sync.rs

```diff
@@ -223,6 +269,74 @@ impl DocCommands {
                     println!("{}", fmt_entry(&entry));
                 }
             }
+            Self::Import {
+                doc,
+                author,
+                prefix,
+                path,
+                in_place,
+            } => {
+                let doc = get_doc(iroh, env, doc).await?;
+                let author = env.author(author)?;
+                let mut prefix = prefix.unwrap_or_else(|| String::from(""));
+
+                if prefix.ends_with('/') {
+                    prefix.pop();
+                }
+                let root = canonicalize_path(&path)?.canonicalize()?;
+                let tag = tag_from_file_name(&root)?;
```

Comment:
We may want this to be `SetTagOption::Auto`, but otherwise the dir name (or file name if the path is to a file) is the tag